.gitlab/corona.yml

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2010-2021, Lawrence Livermore National Security, LLC. Produced
+# at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+# LICENSE and NOTICE for details. LLNL-CODE-806117.
+#
+# This file is part of the MFEM library. For more information and source code
+# availability visit https://mfem.org.
+#
+# MFEM is free software; you can redistribute it and/or modify it under the
+# terms of the BSD-3 license. We welcome feedback and contributions, see file
+# CONTRIBUTING.md for details.
+
+# GitLab pipelines configurations for the Corona machine at LLNL
+
+.on_corona:
+  tags:
+    - shell
+    - corona
+  rules:
+    # Don’t run corona jobs if...
+    # Note: This makes corona an "opt-in" machine. Gitlab instances where
+    # corona jobs should run should set "ON_CORONA" to "ON".
```

Comment:
Some more detailed instructions would be valuable here. I think I figured it out (adding the `ON_CORONA` variable in the GitLab repository settings), but it wasn't the most clear to me.

Maybe also note that this variable in set for our "LGTM" repo. (Incidentally, we really should change the trigger to something else, e.g. "LC GitLab: test")

Reply:
1st point: understood. 👍🏻 
2nd point: I will remind gitlab admins about this. ;) 